backend/service/audit/storage/sql/sql.go

```diff
@@ -246,3 +279,34 @@ func convertResources(proto []*auditv1.Resource) []*resource {
 	}
 	return resources
 }
+
```

Comment:
for `convertAPIBody`, I added a nil check because it can receive a typed nil value from `meta. APIBody`. Similar thought process for checking for nil in `apiBodyProto`

Reply:
thinking on how this flow for processing the metadata can be made simpler so that the nil checks can be removed

Reply:
when input is type proto message with nil value, `protojson.Marshal()` returns slice [123 125] (which translates to empty obj) and err is nil.

when input is nil, `protojson.Unmarshal()` return empty object and err is nil.